typescript/src/electrum.ts

```diff
@@ -253,6 +253,58 @@ export class Client implements BitcoinClient {
     )
   }
 
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   * @see {BitcoinClient#getTransactionHistory}
+   */
+  getTransactionHistory(
+    address: string,
+    limit?: number
+  ): Promise<Transaction[]> {
+    return this.withElectrum<Transaction[]>(async (electrum: Electrum) => {
+      const script = bcoin.Script.fromAddress(address).toRaw().toString("hex")
+
+      // eslint-disable-next-line camelcase
+      type HistoryItem = { height: number; tx_hash: string }
+
+      let historyItems: HistoryItem[] = await this.withBackoffRetrier<
```

Comment:
Nitpicking: Instead of having `await`, I think we could use `.then` to chain promises together.

Reply:
I think `async/await` style leads to more readable code and is generally preferred over `then`. It is also widely used in our codebase and I think mixing both styles is not necessarily the best idea.